### ISSUE TITLE
v0.11.11 on macOS 10.14.3 - show missing -json option

### ISSUE THREAD

**Issue by caseyoneill** [hashicorp/terraform#20544]
### Terraform Version

```
Terraform v0.11.11
+ provider.aws v2.0.0
+ provider.azurerm v1.22.1
```

### Terraform Configuration Files

```hcl
provider "aws" {
  access_key = 
  secret_key = 
  region     = "us-west-2"
}
...
provider "azurerm" {
  subscription_id = 
  client_id       = 
  client_secret   = 
  tenant_id       = 
}

...

```

### Debug Output

```
$ terraform show -json
Usage: terraform show [options] [path]

  Reads and outputs a Terraform state or plan file in a human-readable
  form. If no path is specified, the current state will be shown.

Options:

  -module-depth=n     Specifies the depth of modules to show in the output.
                      By default this is -1, which will expand all.

  -no-color           If specified, output won't contain any color.
```

### Expected Behavior

terraform show works as expected and lists all the details of my configuration. The [online documentation](https://www.terraform.io/docs/commands/show.html) says that there is a -json option for show but it seems to be missing from v0.11.11. When I do add -json to the end of the command, it does not output a json file and instead shows me a help command. I have been able to reproduce this on two other Macs.

**Comment by apparentlymart** [comment]
Hi @caseyoneill,

The `-json` option is added for Terraform v0.12.0; it's not available in Terraform v0.11. Terraform v0.12 is not stable yet, so to try out this feature you'll need to [use the beta1 release](https://www.hashicorp.com/blog/announcing-terraform-0-1-2-beta1), and only use it against experimental infrastructure.

**Comment by caseyoneill** [comment]
Oh, that’s a little weird that beta features would show up in the official production documentation. Others projects tend to either version their documentation or put a version number next to the feature. What if the documentation added “Since 0.12.0” to the -json section?

**Comment by apparentlymart** [comment]
Hi @caseyoneill,

Indeed we don't normally publish beta docs early but we made an exception this time because there's a lot of new config language features to try in the beta and so we wanted to make sure there was a reference for them. We have retained both the 0.12 and 0.11 config docs but the rest of the site is not set up for versions.

Normal docs situation will be restored after the 0.12.0 final release. Sorry for the confusion in the meantime.
